shared/libraries/config_protocol/include/config_protocol/config_client_device_impl.h

```diff
@@ -212,11 +213,9 @@ ErrCode INTERFACE_FUNC GenericConfigClientDeviceImpl<TDeviceBase>::unlock(IUser*
 }
 
 template <class TDeviceBase>
-inline ErrCode INTERFACE_FUNC GenericConfigClientDeviceImpl<TDeviceBase>::isLocked(Bool* locked)
+inline ErrCode INTERFACE_FUNC GenericConfigClientDeviceImpl<TDeviceBase>::forceUnlock()
 {
-    OPENDAQ_PARAM_NOT_NULL(locked);
-
-    return daqTry([this, &locked] { *locked = this->clientComm->isLocked(this->remoteGlobalId); });
```

Comment:
This is probably fine to remove, as there has been no releases in between the 1st locking implementation and this one. It does break compatibility, however.